includes/core/classes/class-rest-api.php

```diff
@@ -251,7 +251,7 @@ protected function events_list_route(): array {
 	 * @return bool True if the parameter is a valid RSVP status, false otherwise.
 	 */
 	public function validate_rsvp_status( $param ): bool {
-		return ( 'attend' === $param || 'attending' === $param || 'not_attending' === $param );
+		return ( 'attend' === $param || 'attending' === $param || 'not_attending' === $param || 'remove' === $param );
```

Comment:
I was thinking of `remove` when I was working on anonymous feature. I ended up just going with `attend` so we don't have to add another status. This can be confusing though for removal. 

I think we can update things a bit and remove `attend` and `remove` and replace it with `unspecified`. I think that word can be used in both cases without confusion. Thoughts?

Reply:
PR to update this https://github.com/GatherPress/gatherpress/pull/525